ArduCopter/GCS_Mavlink.cpp

```diff
@@ -685,14 +687,21 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_long_packet(const mavlink_command_
         // param2 : speed during change [deg per second]
         // param3 : direction (-1:ccw, +1:cw)
         // param4 : relative offset (1) or absolute angle (0)
-        if ((packet.param1 >= 0.0f)   &&
-            (packet.param1 <= 360.0f) &&
-            (is_zero(packet.param4) || is_equal(packet.param4,1.0f))) {
+        if ((packet.param1 < 0.0f) || (packet.param1 > 360.0f)) {
```

Comment:
seems like one (and only one) of these should be  <= or >= , cant allow both 360 and 0.